app/models/manager_refresh/save_collection/base.rb

```diff
@@ -2,16 +2,16 @@ module ManagerRefresh::SaveCollection
   class Base
     class << self
       def save_inventory_object_inventory(ems, inventory_collection)
-        _log.info("Synchronizing #{ems.name} collection #{inventory_collection} of size #{inventory_collection.size} to"\
-                " the database")
+        _log.info("SYNCHRONIZING collection #{inventory_collection} of size #{inventory_collection.size} to"\
```

Comment:
so the yelling is so it's nicer in the log, now it will be
```
SYNCHRONIZING
PROCESSING
PROCESSED
SYNCHRONIZED
```

for the quick peak of where we were spending the time for each IC saving.